libp2p/protocols/pubsub/rpc/messages.nim

```diff
@@ -116,3 +116,32 @@ func shortLog*(m: RPCMsg): auto =
     messages: mapIt(m.messages, it.shortLog),
     control: m.control.get(ControlMessage()).shortLog
   )
+
+proc byteSize*(msg: Message): int =
```

Comment:
Same issue as https://github.com/status-im/nim-libp2p/pull/944#discussion_r1322726849, but I guess we can unify that after merging this PR